cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -81,9 +81,15 @@ public void authorized(HttpRequest httpRequest, HttpResponder httpResponder) thr
       throw new BadRequestException("Missing request body");
     }
 
-    CheckAuthorizedResponse response = new CheckAuthorizedResponse(
-      auth.authorized(request.getEntity(), request.getUser(), request.getActions()));
-    httpResponder.sendJson(HttpResponseStatus.OK, response);
+    if (request.getActions() == null) {
```

Comment:
So it is treated differently if someone doesn't provide actions vs provide an empty array for actions? Both return 200 OK, but one with the body saying "No actions to check for authorization in request". That seems a bit inconsistent.